utils/checkdb.py

```diff
@@ -64,6 +64,7 @@ def check_tile_overlap(db, db_root, strict=False, verbose=False):
     Throw an exception if two tiles share an address
     '''
     mall = dict()
+    tiles_type_done = set()
```

Comment:
This will not work because it doesn't account for tile type mixes.  Instead consider another algorithm.  Examples that are immediately obvious:

- Why does make_tile_mask take a db_root directory?  This implies that it is not using the same prjxray.db.Database or prjxray.grid.Grid object, implying that json and db files are loaded repeatedly
- Can make_tile_mask be split into two parts, one that is tile type specific (e.g. do it once) and then is specialized for differennt tile base addresses and word offsets.?

Reply:
Yeah, that was my concern. Anyway okay, I will investigate and see if I can better optimize the script.